diff --git bin/elasticsearch bin/elasticsearch
index 20dc773..cc3944d 100755
--- bin/elasticsearch
+++ bin/elasticsearch
@@ -82,7 +82,8 @@ if [ "x$ES_INCLUDE" = "x" ]; then
                    /usr/local/share/elasticsearch/elasticsearch.in.sh \
                    /opt/elasticsearch/elasticsearch.in.sh \
                    ~/.elasticsearch.in.sh \
-                   `dirname $0`/elasticsearch.in.sh; do
+                   `dirname $0`/elasticsearch.in.sh \
+                   `dirname $SCRIPT`/elasticsearch.in.sh; do
         if [ -r $include ]; then
             . $include
             break
